### DISCUSSION TITLE
kritical bug: we cannot connect to the teamtalk servers running on a server with ip starts with number 5

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1908] [Q&A]
### Description

when i configgered teamtalk on any server that their ip started with 5, for example 5.215.176.218 or something, i'm faled to connect to it with any other network, then i needed to open a tikit to my server provider to tell them please change my datassenter to make my ip to start with another number, then when they made my ip to start with 141, my teamtalk problem has been fixed: also, my wify ip starts with 5, i'm unable to connect to the tt server i run in my laptop, even if i put the teamtalk ports in the port forwarding list.

### Application

- [X] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by jeremyp3** [comment]
can't reproduce the problem with an ip of 5.215.176.0/24 on my network. the connection to teamtalk works perfectly.

there must be a problem with your teamtalk configuration on this particular network. misconfiguration, banning, etc.

I tested with a server in 5.13.0.5014

**Comment by amirmahdifard** [comment]
@jeremyp3 strange, but what could be the problem? my ports are open, the servis is running, even it works fine with localhost, keep in mind it is windows server i tested with.

**Comment by jeremyp3** [comment]
hm, i tested with a linux server, because that's what i had on hand. 
which version of teamtalk are you experiencing the problem with? 5.13?

**Comment by amirmahdifard** [comment]
i even had this problem with older versions of teamtalk, but yes, last version, 5.13.1.5115
